### DISCUSSION TITLE
Conflict between Nebula and Woocommerce

### DISCUSSION THREAD

**Discussion by FabiolaDErcole** [chrisblakley/Nebula#2130] [Q&A] [answered]
Hi there,

I am using Nebula as a theme for my e-commerce. The thing is that when I click on Appearance --> Customize --> Wordpress the system force the logout from my admin account and I see a white page with the text

You need a higher level of permission.
Sorry, you are not allowed to customize this site.

I tried the same process with the default wordpress theme Twenty Twenty One and it works fine as expected.

Can you please fix this?

Thanks

**Comment by chrisblakley** [comment, marked as answer]
Just going through older issues/discussions and cleaning up– this was resolved with https://github.com/chrisblakley/Nebula/issues/2131
